src/NServiceBus.AcceptanceTests/Routing/MessageDrivenSubscriptions/When_subscribing_with_scaled_out_subscriber_with_distribution_strategy.cs

```diff
@@ -0,0 +1,133 @@
+namespace NServiceBus.AcceptanceTests.Routing.MessageDrivenSubscriptions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using Configuration.AdvanceExtensibility;
+    using EndpointTemplates;
+    using Features;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+    using Settings;
+
+    public class When_subscribing_with_scaled_out_subscriber_with_distribution_strategy : NServiceBusAcceptanceTest
```

Comment:
would it make more sense to name this `When_publishing_to_scaled_out_subscribers_with_distribution_strategy` ?